packages/trpc/server/routers/viewer/webhook/getByViewer.handler.ts

```diff
@@ -80,7 +89,8 @@ export const getByViewerHandler = async ({ ctx }: GetByViewerOptions) => {
     throw new TRPCError({ code: "INTERNAL_SERVER_ERROR" });
   }
 
-  const userWebhooks = user.webhooks;
+  let userWebhooks = user.webhooks;
+  userWebhooks = userWebhooks.filter(filterWebhooks);
```

Comment:
We need to do the same for team webhooks as well

Reply:
Sure